Editor/Helper/LabelAttribute.cs.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 37caba21d7805aa4ebe038621d6ef06a
+guid: e0aeab52b0ca4df46a2646ba90c9376c
```

Comment:
Unnecessary GUID changes

Reply:
And since you created the `Editor/Attributes` folder, this file should be part of it.